sql/sql_lex.cc

```diff
@@ -756,6 +756,9 @@ void LEX::start(THD *thd_arg)
   vers_conditions.empty();
 
   is_lex_started= TRUE;
+
+  seq_already_incr= false;
+
```

Comment:
Remove extra new lines here. There is no reason to single out this particular assignment.